AppConnectSwift/CaptureImageViewController.swift

```diff
@@ -11,14 +11,22 @@ import UIKit
 class CaptureImageViewController: UIViewController, UIImagePickerControllerDelegate, UINavigationControllerDelegate {
     
     @IBOutlet weak var imageView: UIImageView!
+    @IBOutlet weak var saveImage: UIBarButtonItem!
```

Comment:
These two variable names are not great. saveImage more than imageView -- is that a button? 
maybe: "saveImageButton" ?